Core/Core/Views/CoreWebViewController.swift

```diff
@@ -77,7 +77,7 @@ public class CoreWebViewController: UIViewController, CoreWebViewLinkDelegate {
 
         let message = UILabel()
         message.translatesAutoresizingMaskIntoConstraints = false
-        message.text = NSLocalizedString("Interactions on this page are limited by your institution.", comment: "")
+        message.text = NSLocalizedString("Interactions on this page are limited by your institution.", bundle: .core, comment: "")
```

Comment:
Did you run some regex search to find all of these? If so, mind sharing?

Reply:
I searched for `", comment: "`. There were some false positives where we use the `value: ""` parameter, and it may have missed some multiline ones.

Reply:
Smart. thanks!